src/middlewared/debian/control

```diff
@@ -23,6 +23,7 @@ Build-Depends: alembic,
                python3-cryptit,
                python3-dbus,
                python3-dnspython,
+               python3-email-validator,
```

Comment:
Why is this being pulled in? I don't see where it's being used.

Reply:
`pydantic` will crash on attempt to use `EmailStr` if this module is not present